web/src/upload.ts

```diff
@@ -460,19 +476,31 @@ export function setup_upload(config: Config): Uppy<Meta, TusBody> {
 
     uppy.on("upload-success", (file, response) => {
         assert(file !== undefined);
-        let upload_response;
+        if (response.status !== 200) {
+            blueslip.warn("Invalid response status from tus server", {
```

Comment:
Is this really not valid?

Reply:
Not sure of the semantics (if `valid` is the right word or not), but we should be getting a 200 response from the server I'm thinking

Reply:
I think "Tus server returned an error" with details on the error might be fine, but we should not be setting the expectation that it's invalid -- all we know is it's an error.

Reply:
Fixed!